examples/instance_refresh/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13.1"
+
+  required_providers {
+    aws        = ">= 3.22.0"
+    local      = ">= 1.4"
+    null       = ">= 2.1"
+    template   = ">= 2.1"
```

Comment:
We don't need `null` and `template` provider anymore. Please remove them.

Reply:
done